transmart-batch/docs/data_formats/cli-arguments.md

```diff
@@ -3,10 +3,10 @@ Here are command line arguments you could specify for transmart batch upload.
 | Argument | Description                                        | Example                               |
 |----------|----------------------------------------------------|---------------------------------------|
 | -d       | overrides/supplements params file parameter        | `-d STUDY_ID=GSE8581`                 |
-| -p       | specify params file                                | `-p studies/GSE8581/clinical.params`  |
-| -f       | specify folder with params files on any depth      | `-f studies/GSE8581/`                 |
-| -c       | location of database configuration properties file | `-f batchdb.properties`               |
-| -j       | the id or name of a job instance                   |                                       |
+| -p       | specify params file; uploads data specified in file| `-p studies/GSE8581/clinical.params`  |
+| -f       | specify folder; uploads all data recursively       | `-f studies/GSE8581/`                 |
+| -c       | location of database configuration properties file | `-c batchdb.properties`               |
+| -j       | the ID or name of a job instance                   |                                       |
```

Comment:
@wardweistra I have never used the -jrsa parameters. @forus Are these still valid?